yas3fs/__init__.py

```diff
@@ -2830,7 +2852,8 @@ def utimens(self, path, times=None):
             now = get_current_time()
             atime, mtime = times if times else (now, now)
             attr = self.get_metadata(path, 'attr')
-            if attr < 0:
+            logger.debug('utimens attr %s' % attr)
+            if not attr:
```

Comment:
Again, this logic isn't the same; intentional?

Reply:
Same as above